keywords/couchbaseserver.py

```diff
@@ -443,29 +443,27 @@ def create_bucket(self, cluster_config, name, ram_quota_mb=1024, ipv6=False):
             self._create_internal_rbac_bucket_user(name, cluster_config=cluster_config)
 
         # Create client an retry until KeyNotFound error is thrown
-        start = time.time()
         time.sleep(5)
-        while True:
-            if time.time() - start > keywords.constants.CLIENT_REQUEST_TIMEOUT:
-                raise Exception("TIMEOUT while trying to create server buckets.")
-            try:
-                if self.cbs_ssl and ipv6:
-                    connection_url = "couchbases://{}/{}?ssl=no_verify&ipv6=allow".format(self.host, name)
-                elif self.cbs_ssl and not ipv6:
-                    connection_url = "couchbases://{}/{}?ssl=no_verify".format(self.host, name)
-                elif not self.cbs_ssl and ipv6:
-                    connection_url = "couchbase://{}/{}?ipv6=allow".format(self.host, name)
-                else:
-                    connection_url = "couchbase://{}/{}".format(self.host, name)
-                bucket = Bucket(connection_url, password='password')
-                bucket.get('foo')
-            except NotFoundError:
-                log_info("Key not found error: Bucket is ready!")
-                break
-            except CouchbaseError as e:
-                log_info("Error from server: {}, Retrying ...". format(e))
-                time.sleep(1)
-                continue
+        try:
+            if self.cbs_ssl and ipv6:
```

Comment:
retries removed here . Wondering if it is consistently pass all the time without retries. I am concerned that it might fail once in a while without retries.

Reply:
no need of retires, since we have timeout in the api. And this api is been called many time and retry is not added anywhere. This must be the first implementation